administrator/components/com_banners/src/Model/ClientModel.php

```diff
@@ -134,5 +134,18 @@ protected function loadFormData()
     protected function prepareTable($table)
     {
         $table->name = htmlspecialchars_decode($table->name, ENT_QUOTES);
+
+        $app = \Joomla\CMS\Factory::getApplication();
+
+        $existing = $this->getTable();
+        $existing->load(['name' => $table->name]);
+
+        // Show information message
+        if ($existing && $existing->id != $table->id) {
+            $app->enqueueMessage(
+                JText::_('COM_BANNERS_CLIENT_DUPLICATE_INFO'),
```

Comment:
```suggestion
                JText::_('COM_BANNERS_CLIENT_DUPLICATE_INFO'),
```
JText is an old, deprecated class alias which will fail in Joomla 6. Please use `Text` and add a `use` statement for that class to the op.